dev/generate-gapic-services.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
I think this was probably added by accident? Let's not make this script part of this PR. 

If you think we should add this script in the `dev` component, let's do that in a separate PR